### DISCUSSION TITLE
What is the meaning of pct.1 and pct.2 in FindAllMarkers( ) result table?

### DISCUSSION THREAD

**Discussion by shengjun97** [satijalab/seurat#4731] [Q&A] [answered]
Though it seems easy to infer that pct.1 means the percentage of cells highly expressing one certain gene in the designated cluster and pct.2 means the percentage of cells highly expressing the same gene in other clusters, **I wonder the exact meaning of these two column names' meaning**. Because when I need to explain how I find the DEG, I have to explain the rationale for doing so. But it is kind of tricky for me to tell exactly what pct.1 and pct.2 stand for.

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

Using following example
`markers <- FindMarkers(object = obj, ident.1 = "cluster1", ident.2 = "cluster2")`

pct.1 is the percentage of cells expressing the gene/feature in ident.1 and pct.2 is the percentage of cells that express gene/feature in ident.2.

Best,
Sam

**Reply by dd123456000** [reply]
hey！"Why do I get different results after adjusting 'ident1' and 'ident2'?"

**Reply by samuel-marsh** [reply]
Can you explain more?

**Reply by denvercal1234GitHub** [reply]
Hi @samuel-marsh and @saketkc -- I wonder if the `pct` in the results from `FindMarkers` or `FindAllMarkers` are actually "proportions" and not "percentages", because it divides the number of non-zero values by the total number of values, which gives a result between 0 and 1? If so, then the description of Value in ?Help is misleading.

Thank you!
